starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Apple Pie Recipe</title>
+   <meta charset="utf-8">
+   <title>Apple Pie Recipe</title>
+   <link rel="stylesheet" href="./css/main.css"/>
 </head>
+
 <body>
-Apple Pies
+<h1> 
+    <div class="container">
+        <img src="./images/apple-pie.jpg" alt="Apple Pie" style="width:85%; height : 500px">
```

Comment:
If you give your div a ``background-image`` as suggested, you won't need to import the ``<img />`` tag here.